src/main/java/org/spongepowered/api/util/metric/MetricsConfigManager.java

```diff
@@ -66,9 +68,23 @@
      * @return true if metrics gathering plugins have permission to gather
      *         data about this plugin
      */
+    @Deprecated
     default boolean areMetricsEnabled(final Object plugin) throws IllegalArgumentException {
         return this.areMetricsEnabled(Sponge.getPluginManager().fromInstance(plugin)
                 .orElseThrow(() -> new IllegalArgumentException("The supplied object is not a plugin object.")));
     }
 
+    /**
+     * Gets the current <em>global</em> state of collection.
```

Comment:
There should be documentation what each state means.

Reply:
Requested changes have been made.